kernel/src/hil/acifc.rs

```diff
@@ -0,0 +1,40 @@
+//! Interface for direct control of ACIFC
+//!
+//! Author: Danilo Verhaert <verhaert@cs.stanford.edu>
+
+use returncode::ReturnCode;
+
+/// Enum for selecting which edge to trigger interrupts on.
+// #[derive(Debug)]
+// pub enum InterruptMode {
+//     RisingEdge,
+//     FallingEdge,
+//     EitherEdge,
+// }
+
+pub trait Acifc {
+    /// Initialize the ACIFC
+    fn initialize_acifc(&self) -> ReturnCode;
+
+    /// Enable the clock
+    fn enable_clock(&self);
```

Comment:
Genreally clock management doesn't belong directly in the cross-platform HIL. If it _does_ belong here in this particular case, the comment should explain what it's purpose is exactly.